web-gateway/app/controllers/Main.java

```diff
@@ -17,11 +18,19 @@
     private final UserService userService;
     private final FormFactory formFactory;
 
+    //public static boolean showInlineInstruction = true;
+
+    private Configuration config;
```

Comment:
Same comment as above: if you don't use config outside of the constructor, it isn't necessary to create an instance variable.